src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.data.tag;
+/**
+ * Association class of Person and Tag to represent tag changed during the session
+ * 
+ * TODO: Implement functions that in fact add and remove tags. 
+ */
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+    
+    static StringBuilder builder = new StringBuilder();
```

Comment:
Putting the stringBuilder here doesn't follow the diagram, you should declare it as an object in AddressBook class, check #1618.